pkg/kubectl/stop_test.go

```diff
@@ -648,15 +627,15 @@ func TestSimpleStop(t *testing.T) {
 		},
 		{
 			fake: &reaperFake{
-				Clientset:       fake.NewSimpleClientset(service()),
-				noDeleteService: true,
+				Clientset:   fake.NewSimpleClientset(pod()),
+				noDeletePod: true,
 			},
-			kind: api.Kind("Service"),
+			kind: api.Kind("Pod"),
 			actions: []testcore.Action{
-				testcore.NewGetAction(api.Resource("services").WithVersion(""), metav1.NamespaceDefault, "foo"),
+				testcore.NewGetAction(api.Resource("pods").WithVersion(""), metav1.NamespaceDefault, "foo"),
 			},
 			expectError: true,
-			test:        "stop service fails, can't delete",
+			test:        "stop pod fails, can't delete",
```

Comment:
This mocks when apiserver returns success on GET, but returns an error on DELETE request.
The error can be due to many reasons. Some examples:
* invalid request (due to kubectl and apiserver version skew).
* Object deleted by someone else after GET returned but before DELETE request is sent by kubectl.
* Internal server error on the server due to a bug in DELETE logic.